bin/compile

```diff
@@ -60,7 +60,12 @@ set -o xtrace
 
 export PATH="$HOME/.phpenv/bin:$HOME/.php-build/bin:$PATH"
 
-php-build -i development "${VERSION}" "${INSTALL_DEST}/${VERSION}"
+buildDefinition=${VERSION}
+if [[ $VERSION == nightly* || $VERSION == master* ]]; then
+    buildDefinition=8.0snapshot
```

Comment:
There is no need for a new variable `buildDefinition`; you can just override `VERSION`.

Reply:
@BanzaiMan thx i can adjust this, i was not sure if maybe, you wanted to keep the install destination and i was not sure what pushd does, so i went with the change that has the least effect on other usages on version :smile:

Reply:
If my memory serves me right, `php-build …` would create the directory `${INSTALL_DEST}/${VERSION}`; the first argument is for picking the configuration flags.